srv/exo/bin/backup_plateforme.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash -eu
+
+# #############################################################################
+# Initialize
+# #############################################################################                                              
+SCRIPT_NAME="${0##*/}"
+SCRIPT_DIR="$( cd -P "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+SCRIPT_START_TIME=$(date +%s)
+
+# Load env settings
+source ${SCRIPT_DIR}/_setenv-template.sh
+# Load common functions
+source ${SCRIPT_DIR}/_functions.sh
+
+DOWNTIME_START_TIME=$(date +%s)
+# Stop it
+ssh ${EXO_USER}@${EXO_PLF_SERVER} ${SCRIPT_DIR}/_stopPLF.sh
+
+# Dump data
+ssh ${EXO_USER}@${EXO_PLF_SERVER} ${SCRIPT_DIR}/_dumpData.sh
+# Dump database
+ssh ${EXO_USER}@${EXO_DB_SERVER} ${SCRIPT_DIR}/_dumpMysqlDatabase.sh
+# Dump MongoDB
+ssh ${EXO_USER}@${EXO_MONGO_SERVER} ${SCRIPT_DIR}/_dumpMongoDb.sh
+# Dump Elastic
+ssh ${EXO_USER}@${EXO_ES_SERVER} ${SCRIPT_DIR}/_stopElasticSearch.sh
+ssh ${EXO_USER}@${EXO_ES_SERVER} ${SCRIPT_DIR}/_dumpElasticSearch.sh
+ssh ${EXO_USER}@${EXO_ES_SERVER} ${SCRIPT_DIR}/_startElasticSearch.sh
+# Start it
+ssh ${EXO_USER}@${EXO_PLF_SERVER} ${SCRIPT_DIR}/_startPLF.sh
+DOWNTIME_END_TIME=$(date +%s)  
+if [ ${DOWNLOAD_BACKUP} ]; then
```

Comment:
what if the DOWNLOAD_BACKUP is ``false`` ?

If 'm not wrong this option is here to retrieve the backup files from several servers when there is several.
WDYT to rsync the remote dirs ``${EXO_USER}@${EXO_DB_SERVER}:${BACKUP_WORKING_DIR}/...`` when ``DOWNLOAD_BACKUP=true`` and to only rsync the local ``${BACKUP_WORKING_DIR}/...`` when ``DOWNLOAD_BACKUP=false``  ?

Reply:
Done